index.html

```diff
@@ -344,6 +360,12 @@ <h2>The <dfn>PerformanceEntry</dfn> interface</h2>
         duration concept doesn't apply, a performance metric may choose to
         return a `duration` of <code>0</code>.
       </dd>
+      <dt><dfn>navigationId</dfn></dt>
+      <dd>
+        This attribute MUST return the value of the global object's <a>current
```

Comment:
Just return the value it was set to (this is normative, if you want you can add explanation in non normative section)